eclair-core/src/test/scala/fr/acinq/eclair/integration/PaymentIntegrationSpec.scala

```diff
@@ -127,6 +127,8 @@ class PaymentIntegrationSpec extends IntegrationSpec {
           sender.expectMsgType[Iterable[ChannelAnnouncement]].size == channels
         }, max = 60 seconds, interval = 1 second)
         awaitCond({
+
+          
```

Comment:
nit: why the extra lines?

Reply:
I'm cherry picking individual changes from my prototype branch, some artefacts go through 🤷‍♂️

Reply:
Fixed in 35bfff8f3.